zod/data_classes/info.py

```diff
@@ -130,3 +131,8 @@ def get_camera_frames(
 
     def get_lidar_frames(self, lidar: Lidar = Lidar.VELODYNE) -> List[LidarFrame]:
         return self.lidar_frames[lidar]
+
+    def get_radar_frames(self, radar: Radar) -> RadarFrames:
+        if self.radar_frames is None:
+            raise ValueError("No radar frames available.")
```

Comment:
maybe also inform the user that they should download the new dataset (maybe even give them the command here for maximal user friendlyness)? Or can this also happen when the data is downloaded due to some missing files?

Reply:
This shouldn't happen when the data is downloaded. I'll update the message here.

Reply:
nice